features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/modules/statistics/usage.jag

```diff
@@ -259,11 +262,12 @@ function getProviderAPIServiceTime(providerName,from,to, server) {
     }
 }
 
-function getSubscriberCountByAPIs(providerName) {
+function getSubscriberCountByAPIs(providerName, isAllStatistics) {
     var usage = [];
     var usageout = [];
     try {
-        usage = apiProvider.getSubscriberCountByAPIs(providerName);
+        var usageClient=org.wso2.carbon.apimgt.usage.client.UsageClient;
```

Comment:
fixed in commit: https://github.com/wso2/carbon-apimgt/commit/9619da2271b49f37ef93456b6f8a8ac235f685ea